### DISCUSSION TITLE
Vertical spacing between tags. CSS

### DISCUSSION THREAD

**Discussion by RS-JS** [chipzoller/hugo-clarity#217] [Q&A]
Firstly, thank you for taking the time to put together this theme, I think it is great. My question is in regards to the vertical spacing of tags. As you can see on [my site at the moment ](https://robertstjohnsmith.com/) the tags are touching each other. 

I understand I should be able to change this by overwriting the line-height value in the CSS. I can't for the life of me, figure out where I need to change it.  Any help much appreciated. 

Regards

Rob

**Comment by chipzoller** [comment]
Have you happened to get the latest version of the theme?

**Reply by RS-JS** [reply]
Just updated the theme, I still haven't worked out how you can change the spacing.

**Reply by chipzoller** [reply]
Would you mind opening an issue for this? I would regard this as a bug for us to fix.

**Reply by RS-JS** [reply]
sure will do
